mapfishapp/src/main/webapp/app/js/GEOR_mapinit.js

```diff
@@ -340,6 +350,67 @@ GEOR.mapinit = (function() {
                 }
             }
         });
+        
+        // support for JSON search parameter
+        if( initSearch.hasOwnProperty('typename') && initSearch.hasOwnProperty('featurens') 
+            && initSearch.hasOwnProperty('owsurl') && initSearch.hasOwnProperty('cql_filter') ) {
+            var record = {
+                typeName: initSearch.typename,
+                featureNS: initSearch.featurens,
+                owsURL: initSearch.owsurl,
+                WFSversion : "1.1.0"
```

Comment:
I think default value is 1.0.0 : https://github.com/openlayers/ol2/blob/master/lib/OpenLayers/Protocol/WFS.js#L85

So I will replace with 2.0.0 for use the latest version is it OK ?